isaaq/Problem/ProblemGenerator.py

```diff
@@ -53,8 +53,8 @@ def SimplifyCircuit(circuit: QuantumCircuit) -> QuantumCircuit:
 	gates_src: list[BaseGate] = [g for g in circuit.gates]
 	factor: int = 10
 	D = factor
+	gates_dst: list[BaseGate] = []
 	while(D < len(gates_src) * factor):
-		gates_dst: list[BaseGate] = []
```

Comment:
Move this to solve runtime error (variable not defined) : https://github.com/SoshunNaito/ISAAQ/blob/d746290d2f23077346be0478a248d5bbea5e6822/isaaq/Problem/ProblemGenerator.py#L71 🤔